app/javascript/components/restaurants/CuisinesFilter.js

```diff
@@ -0,0 +1,20 @@
+import React from "react"
+
+class CuisinesFilter extends React.Component {
+
+    componentDidMount() {
+
+        fetch('http://localhost:3000/restaurants').then( results => {
```

Comment:
I think there's no need for the prefix
just fetch('/restaurants')

Reply:
Oops.. this file is obsolete and should have been deleted.